integration-tests/cli/resource-deleter.it.js

```diff
@@ -0,0 +1,106 @@
+import fs from 'mz/fs'
+import tmp from 'tmp'
+import { getCredentials } from '@commercetools/get-credentials'
+import { exec } from 'mz/child_process'
+import { version } from '@commercetools/resource-deleter/package.json'
+import {
+  cart,
+  categories,
+  customer,
+  sampleProductType,
+  anotherSampleProductType,
+} from './helpers/resource-delete.data'
```

Comment:
why not also import, create and test the other sample data? I see you created `review` and `payment` for example.